### PR TITLE
Added pre-prio definition

### PR DESCRIPTION
**Pull request by itchingpixels**

### CODE REVIEW

File: meetings/pre-prio.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Pre-prio call
+
+### Responsibilities before the pre-prio:
+
+#### For Contributors:
+Look at what you have planned in the coming week and prepare to 1) highlight if you don’t have enough to do and 2) be able to answer if you can pick up more work.
+
+#### Project/domain owners:
+Look at your project list and prepare to 1) ask for contributors time (i.e. starting something).
```
Comment (lucjac) [inline meetings/pre-prio.md]: Who decides if there is a conflict for contributors time? Project/Domain/Exec owner?

Comment (lucjac) [inline meetings/pre-prio.md]: if there is a 1) is there also a 2) or can we remove it?

Comment (itchingpixels) [reply meetings/pre-prio.md]: fair question!
my proposal would be:
- if the conflict is on the domain level, let the domain owner decide
- if the conflict is between domains, let the exec sponsor decide

I'll add this to the document!

Comment (itchingpixels) [reply meetings/pre-prio.md]: yes:)

Comment (lucjac) [reply meetings/pre-prio.md]: ty!

File: meetings/pre-prio.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### For Contributors:
+Look at what you have planned in the coming week and prepare to 1) highlight if you don’t have enough to do and 2) be able to answer if you can pick up more work.
+
+#### Project/domain owners:
+
+Look at your project list and prepare to:
+1. Ask for contributors time (i.e. starting on something).
+2. Ask for status update (if needed)
+2. Plan what state the project could get to by next week
```
Comment (thomasengebrand) [inline meetings/pre-prio.md]: Should be "3." I think

Comment (itchingpixels) [reply meetings/pre-prio.md]: 😱 yes!

**Pull request merged**